assets/xml/objects/object_door_killer.xml

```diff
@@ -12,6 +12,6 @@
         <DList Name="object_door_killer_DL_001550" Offset="0x1550"/>
         <DList Name="object_door_killer_DL_0017B8" Offset="0x17B8"/>
         <DList Name="object_door_killer_DL_001A58" Offset="0x1A58"/>
-        <Skeleton Name="object_door_killer_Skel_001BC8" Type="Flex" LimbType="Standard" Offset="0x1BC8"/>
+        <Skeleton Name="object_door_killer_Skel_001BC8" Type="Flex" LimbType="LOD" Offset="0x1BC8"/>
```

Comment:
lod type? this seems weird, i dont think the code that draws it agrees. can the non-lod draw functions draw lod limbs?

Reply:
The struct layout for StandardLimb and LodLimb is the same except for an extra `Gfx*` pointer, so I think non-lod draw functions would work on LodLimb. I'm not really sure what to make of this, for these limbs the two display lists are the same so it doesn't seem very LOD-y